packages/edit-site/src/components/post-edit-header/style.scss

```diff
@@ -0,0 +1,9 @@
+.edit-site-post-edit-header {
+	.edit-site-post-edit-header__description {
+		color: $gray-700;
+	}
+
+	.edit-site-post-edit-header__title {
+		padding: 2px 0;
```

Comment:
Is this to align the icon and title? I _think_ we can do that by adjusting the `align` property on the `HStack`, then remove this style.

Reply:
Yeah, and thanks for calling out the use of `align`. Align `center` allows me to remove the above :)